src/__tests__/lib/containers/widgets/query-filter/QueryFilter.test.tsx

```diff
@@ -99,12 +99,25 @@ describe('handling child component callbacks', () => {
     ]
 
     const enumWrapper = wrapper.find('EnumFacetFilter').at(0)
-    enumWrapper.simulate('change', 'Ford', true)
+    enumWrapper.simulate('change', {'Ford': true})
     const expected = _.cloneDeep(lastQueryRequestResult)
     expected.query = { ...expected.query, selectedFacets: expectedResult }
     expect(mockExecuteQueryRequest).toHaveBeenCalledWith(expected)
   })
 
+
```

Comment:
Looks like theres a bunch of unnecessary spaces here

Reply:
Thanks for catching that! Just removed the spacing.